Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -69,6 +69,9 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 % Define realignment settings
 tempnii = xASL_io_ReadNifti(InputPath);
 nFrames = double(tempnii.hdr.dim(5));
+if length(x.EchoTime)>1

```

Comment:
@BeatrizPadrela This could go wrong for non-multi-TE sequences. The `EchoTime` can also be (incorrectly) a vector in enhanced DICOM. So we need to add a boolean like `if bMultiEcho`